mathics/core/symbols.py

```diff
@@ -807,26 +816,40 @@ def __getnewargs__(self):
         return (self.name, self.sympy_dummy)
 
 
+class Constant(Symbol, ExpandOnce):
```

Comment:
About this, my feeling is again that to be "Constant" should be a property of a `Symbol` instead of a daughter class.

Reply:
Yes, we should create a class decorator for this.